packages/readme.md

```diff
@@ -0,0 +1,233 @@
+|-------------|---------|----------|------------|
+| PKG_SOURCE_DIR | -    | no  | Name of the folder to which the source of the software application unpacks. Should only be specified if the source of the software application unpacks to a folder whose name does not start with `$PKG_NAME` |
+| PKG_SOURCE_NAME | -   | no  | Name of the file of the source of the software application. Should only be specified if the source of the software application is not the basename of PKG_URL |
+| PKG_PATCH_DIRS | -    | no  | Change the path for pachtes, to apply to source. Normally the path `./patches` is used. Is this variable is set, the normal path will extended with the value of this variable as a subdirectory: `./patches/${PKG_PATCH_DIRS}` |
+| PKG_NEED_UNPACK | -   | no  | ??? can anyone explane, i have no idea. |
+| PKG_TOOLCHAIN | auto  | no  | Control which of the build toolchains is used. For detailed information, see the [Reference](#toolchain-options). |
+| PKG_BUILD_FLAGS | -   | no  | A Space seperated list of flags, which control often used build modification. Flags can be enabled or disables with a prefixed `+`/`-`. For detailed information, see the [Reference](#build_flags-options). |
+| PKG_PYTHON_VERSION | python2.7 | no | Defines the Python version, which should use. |
+| PKG_IS_KERNEL_PKG | - | no  | Set it to `yes`, for packages which include linux kernel modules |
+
+#### Meson Options
+| Variable    | Default | Required |Description |
+|-------------|---------|----------|------------|
+| PKG_MESON_SCRIPT | $PKG_BUILD/meson.build | no | Meson build file to use |
+| PKG_MESON_OPTS_TARGET | - | no   | Options directly passed to meson |
+
+#### CMAKE Options
+| Variable    | Default | Required |Description |
+|-------------|---------|----------|------------|
+| PKG_CMAKE_SCRIPT | $PKG_BUILD/CMakeLists.txt | no | CMake build file to use |
+| PKG_CMAKE_OPTS_HOST<br>PKG_CMAKE_OPTS_TARGET | - | no | Options directly passed to cmake |
+
+#### Configure Options
+| Variable    | Default | Required |Description |
+|-------------|---------|----------|------------|
+| PKG_CONFIGURE_SCRIPT | $PKG_BUILD/configure | no | configure script to use |
+| PKG_CONFIGURE_OPTS<br>PKG_CONFIGURE_OPTS_BOOTSTRAP<br>PKG_CONFIGURE_OPTS_HOST<br>PKG_CONFIGURE_OPTS_INIT<br>PKG_CONFIGURE_OPTS_TARGET | - | no | Options directly passed to configure |
+
+#### Make Options
+| Variable    | Default | Required |Description |
+|-------------|---------|----------|------------|
+| PKG_MAKE_OPTS<br>PKG_MAKE_OPTS_BOOTSTRP<br>PKG_MAKE_OPTS_HOST<br>PKG_MAKE_OPTS_INIT<br> PKG_MAKE_OPTS_TARGET | - | no | Options directly passed to make in the build step
+| PKG_MAKEINSTALL_OPTS_HOST<br>PKG_MAKEINSTALL_OPTS_TARGET | - | no | Options directly passed to make in the install step
+
+#### Addons
+These options only needed, when the package is build as an addon.
+"Required" column is focus addon packages, only. When the package is no addon, none of these options is required.
```

Comment:
Additional options used when the package builds an addon.

^ only this line is needed